PKGBUILD

```diff
@@ -2,23 +2,27 @@
 pkgname=swaync
 pkgver=0.6.1
 pkgrel=1
-pkgdesc="A simple notificaion daemon with a GTK panel for checking previous notifications like other DE's"
+pkgdesc="A simple notificaion daemon with a GTK panel for checking previous notifications like other DEs"
 _pkgfoldername=SwayNotificationCenter
 url="https://github.com/ErikReider/$_pkgfoldername"
-arch=(x86_64)
+arch=(
+	'x86_64'
+	'aarch64' # ARM v8 64-bit
+	'armv7h'  # ARM v7 hardfloat
+)
 license=(GPL)
 depends=("gtk3>=3.22" "gtk-layer-shell>=0.1" "dbus" "glib2>=2.50" "gobject-introspection>=1.68" "libgee>=0.20" "json-glib>=1.0" "libhandy>=1.4.0")
 conflicts=("swaync" "swaync-client")
 provides=("swaync" "swaync-client")
 makedepends=(vala meson git scdoc)
 source=("${_pkgfoldername}-${pkgver}.tar.gz::${url}/archive/v${pkgver}.tar.gz")
-sha256sums=('SKIP')
+sha256sums=('d762bb4fc723b6efc9f2fe5f1a1eea851fa73a8c5fcac9311fa437809cdf46d4')
 
 build() {
-  arch-meson "${_pkgfoldername}-${pkgver}" build -Dscripting=true
-  ninja -C build
+	arch-meson "${_pkgfoldername}-${pkgver}" build -Dscripting=true
+	ninja -C build
 }
 
 package() {
-  DESTDIR="$pkgdir" meson install -C build
+	DESTDIR="$pkgdir" meson install -C build
```

Comment:
Unnecessary change to tabs. I use `shfmt -ci -s -bn -i 4` to format these pkgbuild files so it's easier to keep it like that :)

Reply:
Was done for accessibility